app/controllers/user_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UserController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+  def index
+    @users = User.all
+  end
+
+  def show
+    puts params
+  end
```

Comment:
* Kindly Make sure that your app handles the URLs used in the wireframes below as stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Take%20a%20look%20at%20the%20wireframes%20below.%20Make%20sure%20that%20your%20app%20handles%20the%20URLs%20used%20in%20the%20wireframes%3A) As when i got to the URLS i get errors 
![image](https://user-images.githubusercontent.com/52497246/184733623-bbaa7be9-a17c-40b9-aa42-ea7dd84bf4da.png)
![image](https://user-images.githubusercontent.com/52497246/184733900-ea44390e-c274-42de-a068-1038309cf5e7.png)